mgmtd/mgmt_be_adapter.c

```diff
@@ -762,8 +765,10 @@ struct msg_conn *mgmt_be_create_adapter(int conn_fd, union sockunion *from)
 	MGMTD_BE_ADAPTER_DBG("Added new MGMTD Backend adapter '%s'",
 			     adapter->name);
 
+#if 0 /* wait until we receive the SUBSCR_REQ registration with name */
 	/* Trigger resync of config with the new adapter */
 	mgmt_be_adapter_sched_init_event(adapter);
+#endif
```

Comment:
Since we moved scheduling INIT somewhere else, do we need to keep this code ?

Reply:
removed old code.